### PR TITLE
blog: add april sec release announcement

### PR DESCRIPTION
**Pull request by RafaelGSS**
Refs: https://github.com/nodejs-private/security-release/issues/5

### CODE REVIEW

File: pages/en/blog/vulnerability/april-2024-security-releases.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+date: 2024-03-27T03:00:00.000Z
+category: vulnerability
+title:  Wednesday, April 3, 2024 Security Releases
```
Comment (richardlau) [inline pages/en/blog/vulnerability/april-2024-security-releases.md]: It probably doesn't matter but there's a double space here.

Comment (richardlau) [reply pages/en/blog/vulnerability/april-2024-security-releases.md]:
```suggestion
title: Wednesday, April 3, 2024 Security Releases
```

**Review by mhdawson — APPROVED** [review]
LGTM

File: pages/en/blog/vulnerability/april-2024-security-releases.md (hunk 1 above)
Comment (RafaelGSS) [reply pages/en/blog/vulnerability/april-2024-security-releases.md]: It was automatically generated. We'll fix it.

**Review by AugustinMauroy — APPROVED** [review]
LGTM !

File: pages/en/blog/vulnerability/april-2024-security-releases.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+date: 2024-03-27T03:00:00.000Z
+category: vulnerability
+title: Wednesday, April 3, 2024 Security Releases
+slug: april-2024-security-releases
+layout: blog-post.hbs
```
Comment (AugustinMauroy) [inline pages/en/blog/vulnerability/april-2024-security-releases.md]: did you use any script to write this ?

**Review by marco-ippolito — APPROVED** [review]

**Comment by richardlau** [comment]
We normally update the website banner for security release announcements.

**Comment by AugustinMauroy** [comment]
@richardlau feel free to open pr

**Comment by richardlau** [comment]
PR to add the banner https://github.com/nodejs/nodejs.org/pull/6578
I also opened an issue in the private repository where the automation is being added, as the updating the banner ought to be done at the same time as the pre and post security announcement blog posts are generated.

**Pull request merged**